GiniPayApiLib/Tests/Classes/SessionManagerMock.swift

```diff
@@ -16,6 +16,8 @@ final class SessionManagerMock: SessionManagerProtocol {
     static let compositeDocumentId = "826626a0-749f-11e2-bfd6-000000000000"
     static let paymentProviderId = "b09ef70a-490f-11eb-952e-9bc6f4646c57"
     static let paymentRequestId = "118edf41-102a-4b40-8753-df2f0634cb86"
+    static let paymentRequesterUri = "ginipay_test://paymentRequester"
```

Comment:
shouldn't be `ginipay-test`?

Reply:
I've copied from the response example, but change it..better to have the same pattern for all paymentRequesterUri